thaomonster/ACNH-Collections
# Feature/search

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds functionality to the search bar.

### Is this a feature or Fix/Refactor?  

Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

I was too in my head and overcomplicated it :(

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/Components/App/App.js` lines: 18-30;
`src/Components/Header/Header.js` lines 4-11, 16-26;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. git fetch this branch
2. run npm start
3. Type a villager name into the search bar

### Steps To Take
>Any questions and possible steps moving forward.

None

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #8
